application/dash_application/views/single_lane_rna.py

```diff
@@ -602,3 +605,12 @@ def all_kits_requested(click):
     )
     def all_library_designs_requested(click):
         return [x for x in ALL_LIBRARY_DESIGNS]
+
+    @dash_app.callback(
+        Output(ids['show-data-labels'], 'value'),
+        [Input(ids['show-all-data-labels'], 'n_clicks')],
+        [State(ids['show-data-labels'], 'options')]
+    )
+    def all_data_labels_requested(click, avail_options):
+        if click is not None:
```

Comment:
Why the check for `None` on click?

Reply:
This callback fires when the page is loaded (`click` is None), meaning that the drop box gets filled in with all values by default. I do not understand why this does not occur with the other "Fill in All` buttons.

Reply:
^ I also noticed this behaviour and am working on a consistent fix (that involves `if click is not None` everywhere)